packages/cli/lib/lib/webpack/webpack-base-config.js

```diff
@@ -29,11 +28,24 @@ module.exports = function (env) {
 	env.manifest = readJson( source('manifest.json') ) || {};
 	env.pkg = readJson( resolve(cwd, 'package.json') ) || {};
 
-	let babelrc = readJson( resolve(cwd, 'old') ) || {};
-	let browsers = env.pkg.browserslist || ['> 0.25%', 'IE >= 9'];
-
 	let nodeModules = resolve(cwd, 'node_modules');
 
+	// @see https://jamie.build/last-2-versions
+	let browsers = env.pkg.browserslist || ['>0.25%', 'not ie 11', 'not op_mini all'];
+	let babelrc = readJson( resolve(cwd, '.babelrc') ) || {};
+	let babel = require('@preact/babel')(env, { browsers });
+	babel.babelrc = false; // we've already read it
+
+	// support `async` – CLI feature, not Preact-specific
+	babel.plugins.push(
+		[require.resolve('fast-async'), { spec: true }]
+	);
+	babel.presets.forEach(x => {
+		if (Array.isArray(x) && x[0].includes('@babel/preset-env')) {
+			x[1].exclude.push('transform-async-to-generator');
+		}
+	});
```

Comment:
also not sure if this is the best idea, especially if someone wants to extend the config and then async breaks for them

Reply:
If they're overwriting, it gets lost anyway. The extend/overwrite happens _after_ this point.. same as before

Reply:
but if you extend the config you won't have `fast-async` because that will override the default one here

Reply:
Fast async isn't included in preact preset because it's not a part of getting Preact to run. It's a CLI feature not a preact requirement

Reply:
yes, but now if someone makes a custom `.babelrc` and extends from `@preact/babel` they'll get errors

Reply:
Why? This writes into `@preact/babel` 100% of the time.

If someone declares a `.babelrc` file to overwrite what PreactCLI offered, that was ***already*** going to be the case.

Not changing (or looking to change) any logic here. If it should be changed, it should be separate from this PR.

Reply:
hmm, makes sense I guess

but they have to completely override `preset-env` to get it back

Reply:
As of now, that's their choice. And is the current behavior

Reply:
in theory you are changing the behaviour from before though, `fast-async` was in the extendable config from the cli

Reply:
With this package, one can do this:

```js
{
  "presets": [
    "@preact/babel",
    ["@babel/preset-env", {
      "exclude": [
        "transform-async-to-generator"
       ]
     }]
  ],
  "plugins": [
    ["fast-async", { "spec": true }]
  ]
}
```

And they'd be at the same settings. In ^this state, it's a pointless `.babelrc` file, but ofc this assumes the user had something worth doing.

Reply:
is that gonna take the rest of the `preset-env` config from `@preact/babel`?

Reply:
so the only thing I want to know is if your example `.babelrc` would basically get turned into this by babel

```js
["@babel/preset-env", {
	"loose": true,
	"modules": false,
    "targets": {
		"browsers": [
			">0.25%",
			"not ie 11'"
			"not op_mini all"
		]
	}
	"exclude": [
		"transform-regenerator",
		"transform-async-to-generator",
		"transform-typeof-symbol"
	]
}]
```

and our config doesn't get completely overriden